src/sentry/lang/native/unreal.py

```diff
@@ -8,7 +8,7 @@
 import uuid
 
 _portable_callstack_regexp = re.compile(
-    r'((?P<package>[\w]+) )?(?P<baseaddr>0x[\da-fA-F]+) \+ (?P<offset>[\da-fA-F]+)')
+    r'((?P<package>[\w]+) )(?P<baseaddr>0x[\da-fA-F]+) \+ (?P<offset>[\da-fA-F]+)')
```

Comment:
_nit_: These parens are redundant. You could write as:

```suggestion
    r'(?P<package>[\w]+) (?P<baseaddr>0x[\da-fA-F]+) \+ (?P<offset>[\da-fA-F]+)')
```